frontend/public/components/_service.scss

```diff
@@ -15,6 +15,13 @@
   }
 }
 
+.service-hostname {
```

Comment:
I don't think we want it indented. How does it look without this style?

Reply:
<img width="816" alt="Screenshot 2021-09-02 at 17 17 49" src="https://user-images.githubusercontent.com/1668218/131870852-1a3378f3-f9c2-4e02-ae6f-b2e19b076d65.png">

Reply:
I think it's better without the indentation.